styles.css

```diff
@@ -0,0 +1,555 @@
+  color: var(--Primary);
+  flex: none;
+  flex-grow: 0;
+  max-width: 750px;
+}
+
+.social {
+  gap: 20px;
+  margin-top: 3%;
+}
+
+.next {
+  margin-top: 5%;
+  display: flex;
+}
+
+.head {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--Primary);
+}
+
+.img-multi {
+  background-image: url(assest/img/recent/holder.png);
+  background-size: cover;
+  height: 260px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  flex-direction: column;
+  border: 1px solid #ebf0ee;
+  border-radius: 10px;
+}
+
+.text {
+  margin-bottom: 25px;
+  flex-direction: column;
+}
+
+.work-design {
+  font-size: 24px;
+  color: #3a4a42;
+  font-weight: 600;
+}
+
+.tags ul {
+  flex-direction: row;
+  list-style: none;
+  column-gap: 5px;
+  padding: 5px;
+  margin-bottom: 20px;
+}
+
+.tag {
+  background: #ebf0ee;
+  padding: 5px;
+  border: 1px solid #fff;
+  margin-bottom: 5px;
+  border-radius: 5px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+  color: #3a4a42;
+}
+
+.see-project {
+  background: var(--Secondary);
+  border: none;
+  color: #fff;
+  border-radius: 5px;
+  padding: 5px 12px;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.see-project:hover {
+  background-color: #008552;
+  cursor: pointer;
+}
+
+.contant-1 {
+  flex-direction: column;
+  position: relative;
+  gap: 24px;
+  text-align: center;
+}
+
+.get {
+  background: #36b37f;
+  border: none;
+  padding: 9px 19px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  border-radius: 5px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #fff;
+}
+
+.get:hover {
+  background-color: #008552;
+  cursor: pointer;
+}
+
+.tager {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+  justify-content: center;
+}
+
+.card-about {
+  background-color: #ebf0ee;
+  height: 352px;
+  width: 327px;
+  flex-direction: column;
+  border: 1px solid #c1c7d0;
+  border-radius: 10px;
+}
+
+.lang {
+  height: 70px;
+  width: 70px;
+  background-color: #fff;
+  border-color: 1px rgba(255, 255, 255, 0.363);
+  padding: 10px;
+  border-radius: 100px;
+  margin-top: 15%;
+}
+
+.text-1 {
+  margin-bottom: 120px;
+  flex-direction: column;
+}
+
+h4 {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #344563;
+}
+
+.tag-1 {
+  background: #fff;
+  padding: 5px;
+  border: 1px solid #fff;
+  color: #36b37f;
+  border-radius: 5px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+  margin-top: 8%;
+}
+
+.tags-1 ul {
+  flex-direction: row;
+  column-gap: 5px;
+  padding: 5px;
+}
+
+.tags-2 ul {
+  flex-direction: row;
+  column-gap: 5px;
+  padding: 5px;
+}
```

Comment:
- [ ] [OPTIONAL] Your padding of `5px` seem not to be consistent with the `12px` found in the requirement. Spacing is an important concept for user experience. It also makes your user interface really clear for all readers to interact.